server/channels/app/command.go

```diff
@@ -485,6 +485,9 @@ func (a *App) DoCommandRequest(rctx request.CTX, cmd *model.Command, p url.Value
 	ctx, cancel := context.WithTimeout(context.Background(), time.Duration(*a.Config().ServiceSettings.OutgoingIntegrationRequestsTimeout)*time.Second)
 	defer cancel()
 
+	// TODO: check if the URL is contained in any outgoing oauth connection.
+	// If so, authenticate using the first matching connection to retrieve the access token.
```

Comment:
Was the TODO meant for this PR? Otherwise I'd expect a ticket to be created and linked in the comment.

Reply:
Oh yeah, I forgot about this because I wasn't sure if we should cover slash commands in this first iteration. cc: @mickmister